### DISCUSSION TITLE
Update content type - migrate field values

### DISCUSSION THREAD

**Discussion by MikeKry** [OrchardCMS/OrchardCore#9483] [Q&A]
Hi, 

I would like to know, if there is any possibility, to migrate data in existing items from field A to field B.

e.g. when I create new version of widget, where I added new field A and removed some other field B, I would like to transfer data from B to A, so i would not need to do it manually

**Comment by deanmarcussen** [comment]
What I tend to do is export the item to json, edit it, and reimport with different version numbers.

What would be ideal is a feature where this process could be mapped, i.e. source to dest with some drag and drop, and js functionality.

An even better version would handle different sources, i.e. json / csv etc
